tap_github/tests/test_tap.py

```diff
@@ -1,25 +1,17 @@
-import datetime
-
 import pytest
-from singer_sdk.helpers._catalog import (
-    deselect_all_streams,
-    set_catalog_stream_selected,
-)
 
 from tap_github.tap import TapGitHub
 
 from .fixtures import repo_list_config
 
-repo_list_2 = ["octocat/hello-world", "MeltanoLabs/tap-github", "mapswipe/mapswipe"]
+repo_list_2 = ["MeltanoLabs/tap-github"]
```

Comment:
Why the switch to one project?

Reply:
the new collaborators stream needs authentication. Since CI only has access to MeltanoLabs, we can't test the other projects easily anymore as CI doesn't have access to them. What do you reckon we do?

Reply:
We're having the same issue with `tap-gitlab`, but no good solution has come up yet.

Reply:
@laurentS and @Ry-DS 

Couple questions related to auth and CI tokens:

1. With the current CI token, can we have plural projects here if all are from MeltanoLabs (for instance, adding tap-gitlab and one or two others)?
2. With the current CI token, can we get data from the `MeltanoLabs` organization when pulling the org-related streams?

If "no" to either of these, can we get granular enough control using personal access tokens, or do we need to consider a new "bot" user with the proper grants to make the above two work? Or perhaps there's another path forward?

Reply:
Checked what's possible with personal tokens, and we can definitely read org data with them: 
![image](https://user-images.githubusercontent.com/25571542/160285026-5d69b80c-ac3e-462a-8054-6f954bb9863a.png)

Reply:
To address 1. yes, we can put multiple repos in the config, all from MeltanoLabs if needed. I can't think of any problems with this in CI.

Reply:
So maybe the easiest path forward is for you (@aaronsteers ) or someone else on the MeltanoLab org to add a personal access token to this CI?

Reply:
`ORG_LEVEL_TOKEN` is now available 🙂